### PR TITLE
PODS-2228 : PSA variances - Create /Update existing controller in backend and add new end point

### PR DESCRIPTION
**Pull request by abhijeetardale**
PODS-2228 : PSA variances - Create /Update existing controller in backend and add new end point

### CODE REVIEW

File: app/service/SchemeServiceImpl.scala [hunk 1]
```diff
@@ -25,26 +25,47 @@ import play.api.Logger
...
+    implicit headerCarrier: HeaderCarrier, ec: ExecutionContext, request: RequestHeader): Future[Either[HttpException, JsValue]] = {
+
+    getPensionSchemeAdministrator(json) { pensionSchemeAdministrator =>
+      val psaJsValue = Json.toJson(pensionSchemeAdministrator)
+      Logger.debug(s"[PSA-Variation-Outgoing-Payload]$psaJsValue")
+      desConnector.updatePSA(psaId, psaJsValue)
+    }
+  }
+
+  def getPensionSchemeAdministrator(json: JsValue)(
```
Comment (grantrobson) [inline app/service/SchemeServiceImpl.scala]: Can we call this something like convert and make return value Unit explicit?

Comment (grantrobson) [inline app/service/SchemeServiceImpl.scala]: This should be private

Comment (abhijeetardale) [reply app/service/SchemeServiceImpl.scala]: its not only doing conversion but also handling Try

Comment (abhijeetardale) [reply app/service/SchemeServiceImpl.scala]: fixed

**Pull request merged**